ruby-challenges.rb

```diff
@@ -1,16 +1,25 @@
-# ASSESSMENT 5: Ruby Coding Practical Questions
-# MINASWAN
 
-# --------------------1) Create a method that takes in an array of words and a single letter and returns all the words that contain that particular letter.
 
 beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
-
 letter_o = 'o'
 # Expected output: ['coffee', 'soda water']
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
 
+def has_letter(arr, letter)
+  arr.map do |c|
```

Comment:
Remember to more descriptive when naming our arguments, parameters so that it's easy for u and other devs to read.